hook-tests/001-extra-packages.disabled

```diff
@@ -134,3 +134,6 @@ if [ -n "$DIFF" ]; then
     echo "test_pkg_removal.sh test."
     exit 1
 fi
+
```

Comment:
I think we should re-enable this test...

Reply:
Excellent point,  started in https://github.com/snapcore/core-base/pull/102

Reply:
Thanks, I rebased on your PR